CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Azure Kubernetes Service Changelog
 
+
+## Release 2024-04-11
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+- [Dual-stack networking (IPv4/IPv6) with CNI Overlay](https://learn.microsoft.com/en-us/azure/aks/azure-cni-overlay?tabs=kubectl#dual-stack-networking) is now generally available (GA) for Linux node pools and does not need the Azure Feature Flag registered on the subscription.
+
+### Release Notes
+
+* Features:
+
+
+* Preview Features:
+
+
+
+
+* Behavioral Changes:
+-   Clusters running Kubernetes 1.29 or later will have kubernetes.azure.com/managedby=aks  label to tigera-operator deployment in Calico clusters 
```

Comment:
nit: extra spaces before Clusters. Not sure this is a behavioral change though. I know we'll introduce a new label, but when this update rolls out, there shouldn't be any behavior change. or we need to reframe it to capture what might change/break. leaving a note internally on our chat to discuss this